backend/app/repositories/user_repository.py

```diff
@@ -109,22 +110,21 @@ async def update_user(
     return _to_user_entity(user)
 
 
-async def get_user_list_from_organization(
-    user: User, session: AsyncSession
-) -> Union[list[User], User]:
-    """
+async def get_user_list_by_org(
+    user: UserEntity, session: AsyncSession
+) -> Union[List[UserEntity], UserEntity]:
```

Comment:
```suggestion
) -> List[UserEntity]:
```

don't make things more complicated than they have to be: just always return a list (see comment below)